### ISSUE TITLE
ERROR: Invalid codec guid

### ISSUE THREAD

**Issue by nashsu** [JRF63/desktop-streaming#3]
windows 11, compile and run old branch, when click play button, there is an error:

<img width="875" alt="Screenshot 2024-09-17 at 19 21 28" src="https://github.com/user-attachments/assets/fe73f2c2-19c1-4b91-8fdb-8404b5c58562">

please help , thx

**Comment by JRF63** [comment]
Probably AV1? Is this on a RTX 4000 series GPU?

The newer branch can handle this but sadly I haven't had time to work on it recently.

**Comment by nashsu** [comment]
Yes it's on RTX4090, so it's codec problem?  I'll have a look at new dev branch, see if   I can do some codec compatible work.

**Comment by mostofreedom** [comment]
> Yes it's on RTX4090, so it's codec problem? I'll have a look at new dev branch, see if I can do some codec compatible work.

you tested new branch?
